tests/integration/testdata/buildcmd/terraform/image_based_lambda_functions_local_backend/main.tf

```diff
@@ -0,0 +1,90 @@
+provider "aws" {
+    region = "us-west-1"
```

Comment:
we create buckets in us-east-1 in our own integration test class code, do these 2 being different not matter?

Reply:
yeah, it does not matter. The buckets being created are for storing the backend state file generated by `terraform`, which is not related where the resources in the `tf` files are deployed.